src/overlays/Modal.tsx

```diff
@@ -49,6 +55,28 @@ const modalCSS = css`
       transform: translateX(0);
     }
   }
+
+  &.ac-modal--modal {
+    border: 1px solid ${theme.colors.gray500};
+    border-radius: 5px;
+    box-shadow: -10px 0px 30px 10px rgba(0, 0, 0, 0.1);
+    /* Start offset by the animation distance */
+    transform: translateX(500px);
```

Comment:
I think animating in the modal makes sense but not from off screen and should come from the bottom. See https://react-spectrum.adobe.com/react-spectrum/Dialog.html

Reply:
Makes sense! Fixed it to match Dialog here